lib/parallel_tests/grouper.rb

```diff
@@ -28,7 +28,31 @@ def in_even_groups_by_size(items, num_groups, options = {})
           raise 'Number of isolated processes must be less than total the number of processes'
         end
 
-        if isolate_count >= 1
+        if options[:specify_groups]
+          specify_spec_processes = options[:specify_groups].split('|')
```

Comment:
all `_spec_` -> `_test_`
since this is a general base class and not only rspec

Reply:
maybe just do the `,` split here too so there is only 1 value going into the function
`options[:specify_groups].split('|').map { |group| group.split(',') }` and then to get all use `all.flatten(1)`

Reply:
i edited this a bit in my subsequent commit thats coming, let me know what you think